tests/airgapped/airgap.sh

```diff
@@ -79,6 +79,31 @@ function setup_microk8s() {
   airgap_wait_for_pods "$NAME"
 }
 
+function create_ingress_object_in_kubeflow_namespace() {
+  local NAME=$1
+  if ! lxc exec "$NAME" -- bash -c 'cat <<EOF | microk8s kubectl apply -n kubeflow -f -
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: http-ingress
+  namespace: kubeflow
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: istio-ingressgateway-workload
```

Comment:
Is there a programmatic way we can get this value ?

Reply:
I can't think of a simple way to do this before deploying CKF, let's say we wanted to get it from the istio repo where the [service name](https://github.com/canonical/istio-operators/blob/main/charms/istio-gateway/src/manifest.yaml#L296) is, we'd need to render it with the `kind` which is [defined in the bundle](https://github.com/canonical/bundle-kubeflow/blob/main/releases/1.8/stable/kubeflow/bundle.yaml#L41).

Another option is to move this step to be post-deployment after running the deployment script, and programmatically find the service of `LoadBalancer` type. Although I'm not sure where this post-deployment step would live.

As a quick fix I can add a comment that this name has to be the same as the one defined in the charm in the corresponding 1.8 track branch for istio, since this script is specific to 1.8 and we are not yet fully autonomous.

WDYT?

Reply:
Yeah, I think the comment will help out figuring out where the name comes from.

I guess since these scripts are not quite flexible, we can leave it as is for now. Thanks for the clarification.